test/fixtures/operator/installation.yaml

```diff
@@ -0,0 +1,19 @@
+apiVersion: operators.coreos.com/v1
+kind: OperatorGroup
+metadata:
+  name: snyk-operator
+  namespace: snyk-monitor
+spec:
+  targetNamespaces:
+  - snyk-monitor
+---
+apiVersion: operators.coreos.com/v1alpha1
+kind: Subscription
```

Comment:
I really have no idea what this means, but it looks professional

Reply:
Haha, that is what OpenShift does under the hood when you click to install the Operator through the UI!